.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
+
+language: r
+
+cache: packages
+
+r: 3.3.3
+
+r_binary_packages:
+    - devtools
+    - roxygen2
```

Comment:
I recall you saying you wanted to pull packages from Bioconductor in production, to better guarantee compatibility. If that's the case, could you please set up the Travis environment to pull from Bioconductor? We want it to mimic a production environment as closely as possible.

Reply:
I changed my mind on that for two reasons:
* I can specify version dependencies myself now using the DESCRIPTION file. I am taking the versions that Bioconductor 3.6 is using
* I have to use a newer version on Plotly, which is unavailable on either Bioconductor or CRAN. The newer version is a lot more stable.

Since Acquacotta is an R package now, I can enforce package compliance on my machine and on the production machine by running devtools::install()